packages/keycloak-user-management/src/components/forms/UserForm/tests/index.test.tsx

```diff
@@ -336,8 +337,8 @@ describe('components/forms/UserForm', () => {
       await flushPromises();
       wrapper.update();
     });
-    expect(wrapper.find('FormItemInput').at(4).prop('id')).toEqual('practitionerToggle');
-    expect(wrapper.find('FormItemInput').at(4).props()).toMatchSnapshot('practitionerToggle');
+    expect(wrapper.find('FormItemInput').at(5).prop('id')).toEqual('practitionerToggle');
+    expect(wrapper.find('FormItemInput').at(5).props()).toMatchSnapshot('practitionerToggle');
```

Comment:
would adding `id`'s to the formItems make the find less prone to refactors due to form changes